persistence-modules/r2dbc/src/test/resources/application.yml

```diff
@@ -1,6 +1,31 @@
 # R2DBC Test configuration
 r2dbc:
   url: r2dbc:h2:mem://./testdb
+
+server:
+  port: 8080
+
+spring:
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
we might be able to get rid of this property if we let spring handle creating the connection factory and flyway configs as per my recommendation of using different profiles for the configs between postgres and h2.

Reply:
sure, this is updated now